tests/how_to_test_full_app.rs

```diff
@@ -0,0 +1,113 @@
+use bevy::prelude::*;
```

Comment:
Module level docs please :) Laying out a bit of an intro of "this is what we're testing and why" is really helpful.

Reply:
Will do